cmd/stop.go

```diff
@@ -36,13 +37,17 @@ func stopNetwork(cmd *cobra.Command, args []string) error {
 	if len(args) > 0 {
 		snapshotName = args[0]
 	} else {
-		snapshotName = defaultSnapshotName
+		snapshotName = constants.DefaultSnapshotName
```

Comment:
Are we overriding the default snapshot here? What if we cleanup the snapshot directory, or it is gone, the doesn't this mean that we loose the first snapshot default from the installation? Resulting the in the tool not being usable anymore?

Reply:
yes, stop is supposed to overwritte the default snapshot if not given args. if the snapshot directory is deleted by some way (currently the tool does not do that), it will be recreated on init (or where that first copy will do). 
really default snapshot and bootstrap snapshot (the one in the tar) coincide when the tool starts, or after the hidden snapshots dir or default snapshot subdir is deleted.
after that, subsecuent stop will change the default snapshot, to save users work on top of the original one